src/components/Comments/Comments.style.css

```diff
@@ -0,0 +1,14 @@
+.comments-list {
+    list-style: none;
+    text-align: center;
+    padding: 3rem;
+    border: 2px solid #000000;
+    
+  }
+  
+  .comments-header {
+    background-color:#faeee7;
+    text-align: center;
+    padding: 3rem;
+    border: 2px solid #000000;
+  }
```

Comment:
Have another look at the mobile styling of this page, a lot of the nav bar is cut off and the alignment is totally to the side. This is the same for most of your app pages if not all:

<img width="397" alt="image" src="https://user-images.githubusercontent.com/61753159/193237629-b6194dc7-2fc8-4ceb-9e7e-37ba05d94fd8.png">


<img width="604" alt="image" src="https://user-images.githubusercontent.com/61753159/193237545-a6d696a5-1891-45fd-a379-e173cee14b3e.png">